src/js/modules/media/actionTypes.js

```diff
@@ -0,0 +1,5 @@
+export const FETCH_MEDIA_PENDING = 'media/FETCH_MEDIA_PENDING';
+export const FETCH_MEDIA_REJECTED = 'media/FETCH_MEDIA_REJECTED';
+export const FETCH_MEDIA_FULFILLED = 'media/FETCH_MEDIA_FULFILLED';
+export const ADD_MEDIA = 'media/ADD_MEDIA';
+export const MATCH_MEDIA = 'media/MATCH_MEDIA';
```

Comment:
ADD_MEDIA should be something like PROCESS_MEDIA_RESPONSE (don't use that name, actually think about a good one)